### PR TITLE
Full Mission Sets - improve fullmissionsetfolder

### PR DESCRIPTION
**Pull request by PervonHarke**
**When merged this pull request will:**
- change all file names in the full mission folder to adhere to standard naming format
- update deu_2023_pzgren_full_mission_set_v5 to fix error in KoscherGarage file
- update deu_2023_fschjg_full_mission_set_v5 to fix error in KoscherGarage file
- update deu_2023_pztrp_full_mission_set_v5 to add description in warno file

### CODE REVIEW

**Comment by PervonHarke** [comment]
@Tapawingo how do I add a custom milestone? I would add one for having all loadouts be updated to cc modpack v5 compatability.

**Comment by Tapawingo** [comment]
> @Tapawingo how do I add a custom milestone? I would add one for having all loadouts be updated to cc modpack v5 compatability.

https://github.com/clustermod/Harks-CMF3-Loadouts/milestones

File: Full Mission Sets/deu_2023_pzgren_flecktarn_full_mission_set_v5/deu_2023_pzgren_garage_flecktarn_v5.sqf [hunk 1]
```diff
@@ -2,8 +2,8 @@
  * Made for Arma 3 Virtual MILSIM
  *
  * Author: Hark
- * CC Modpack Version: 
- * [Description]
+ * CC Modpack Version: V5
+ * [Logistics Template made for ]
```
Comment (Tapawingo) [inline Full Mission Sets/deu_2023_pzgren_flecktarn_full_mission_set_v5/deu_2023_pzgren_garage_flecktarn_v5.sqf:6]: Incomplete Description.

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf
Comment (Tapawingo) [inline Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf:1]: Empty. Is this intended?

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_loadout_flecktarn_v5.sqf
Comment (Tapawingo) [inline Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_loadout_flecktarn_v5.sqf:1]: Empty. Is this intended?

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_logistics_flecktarn_v5.sqf
Comment (Tapawingo) [inline Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_logistics_flecktarn_v5.sqf:1]: Empty. Is this intended?

Comment (PervonHarke) [reply Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_logistics_flecktarn_v5.sqf:1]: yes

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_loadout_flecktarn_v5.sqf
Comment (PervonHarke) [reply Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_loadout_flecktarn_v5.sqf:1]: no

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf
Comment (PervonHarke) [reply Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf:1]: no

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_loadout_flecktarn_v5.sqf
Comment (Tapawingo) [reply Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_loadout_flecktarn_v5.sqf:1]: Oki, submit a commit that rectifies this and I will resolve this issue :)

File: Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf
Comment (Tapawingo) [reply Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf:1]: Oki, submit a commit that rectifies this and I will resolve this issue :)

Comment (PervonHarke) [reply Full Mission Sets/deu_2023_pztrp_flecktarn_full_mission_set_v5/deu_2023_pztrp_garage_flecktarn_v5.sqf:1]: I tried fixing this but now it threw me into a endless loop of merge conflicts that I cannot resolve and neeeding to "Clean up the repository working tree before checkout"...
It just feels like I am soft locked. Will send more info to you on Discord.

**Pull request closed without merging**